inc/template.php

```diff
@@ -1639,6 +1639,9 @@ function tpl_actiondropdown($empty = '', $button = '&gt;') {
         'user_tools' => array('login', 'register', 'profile', 'admin'),
     );
 
+    $evt = new Doku_Event('TEMPLATE_MOBILETOOLS_DISPLAY', $action_structure);
+    if($evt->advise_before()) $action_structure = $evt->data;
```

Comment:
Sorry for annoying again with code formatting, but could you use { ... }, and move the statement to the new line? Thanks.